docs/CHANGELOG.md

```diff
@@ -9,6 +9,124 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+#### New env vars
+
+`BLOCK_EMISSION_IDLE_WARNING_THRESHOLD` - controls global override for the time after which node will start logging warnings if no heads are received.
+`EVM_DISABLED` - will prevent any chains from being loaded at all, and any jobs that interact with the EVM cannot be created/loaded/run. Distinct from `ETHEREUM_DISABLED` which continue to work as before and will load chains, but will not connect to any nodes and still allows you to create jobs. This is a new recommended setting (replacing `ETH_DISABLED`) for jobs that don't need the EVM at all, e.g. Cron jobs on DYDX.
+`ETH_DEFAULT_BATCH_SIZE` - EvmDefaultBatchSize controls the number of receipts fetched in each request in the EvmConfirmer
+`CLOBBER_NODES_FROM_ENV` - Defaulting to true, this env var when set will autocreate database rows for chain and nodes. It will upsert a new chain using the DefaultChainID and upsert nodes corresponding to the given ETH_URL and ETH_SECONDARY_URLS. It is recommended, after the initial population, to set this env var to false and thereafter to use the CLI commands or API to manage chains/nodes.
+
+#### Multichain support added
+
+Chainlink now supports connecting to multiple different EVM chains simultaneously.
+
+This means that one node can run jobs on Goerli, Kovan, BSC and Mainnet (for example).
+
+Extensive efforts have been made to make this as seamless as possible. Generally speaking, you should not have to make any changes when upgrading your existing node to this version. All your jobs will continue to run as before.
+
+The overall summary of changes is such:
+
+##### Chains/Ethereum Nodes
+
+EVM chains are now represented as a first class object within the chainlink node. You can create/delete/list them using the CLI or API, e.g.
+
+###### Creation
+
+```bash
+chainlink chains evm create -id  # creates an evm chain with chain ID 42 (see: https://chainlist.org/)
+chainlink nodes create -name 'my-primary-kovan-full-node' -type primary -ws-url ws://node.example/ws -http-url http://node.example/rpc # http-url is optional but recommended for primaries
+chainlink nodes create -name 'my-send-only-backup-kovan-node' -type sendonly -http-url http://some-public-node.example/rpc
```

Comment:
by "sendonly" we mean like broadcast transactions only right? could be clarified